core/common/src/main/java/alluxio/util/io/PathUtils.java

```diff
@@ -218,16 +222,25 @@ public static String concatUfsPath(String base, String path) {
   /**
    * Get temp path for async persistence job.
    *
+   * @param ufs the ufs
    * @param path ufs path
    * @return ufs temp path with UUID
    */
-  public static String getPersistentTmpPath(String path) {
+  public static String getPersistentTmpPath(@Nullable UnderFileSystem ufs, String path)
```

Comment:
Instead of a nullable UFS object, how about passing in a configuration object instead?
If you have got a `UnderFileSystem` object, it always has a accompanying `UnderFileSystemConfiguration` object.

```suggestion
  public static String getPersistentTmpPath(UnderFileSystemConfiguration ufsConf, String path)
```